src/lib/GraphicsContext.js

```diff
@@ -1,72 +0,0 @@
-'use strict';
```

Comment:
Yes. If we need in future we can resurrect. I did not want to spend time on making it right now.